src/features/intercom/readme.md

```diff
@@ -0,0 +1,16 @@
+## Intercom
+
+This feature allow us communicate with users by integrated in application chat
+
+### How to use
+
+```ts
+import('~features/intercom').then(({ initIntercom }) => {
+  initIntercom();
+});
+```
+
+### How it works
+
+`initIntercom` function check permission for use cookies that intercom require.
+If it allowed - we set intercom related cookie, settings, and execute intercom sniped that integrate chat in application
```

Comment:
If it **is** allowed - we set intercom related cookie, settings, and execute intercom **snippet** that integrate chat in application